src/style.css

```diff
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-of-type {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-of-type(1),
+.stars--2 .stars__star:nth-of-type(2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-of-type(1),
+.stars--3 .stars__star:nth-of-type(2),
+.stars--3 .stars__star:nth-of-type(3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-of-type(1),
+.stars--4 .stars__star:nth-of-type(2),
+.stars--4 .stars__star:nth-of-type(3),
+.stars--4 .stars__star:nth-of-type(4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-of-type(1),
+.stars--5 .stars__star:nth-of-type(2),
+.stars--5 .stars__star:nth-of-type(3),
+.stars--5 .stars__star:nth-of-type(4),
+.stars--5 .stars__star:nth-of-type(5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
seems it is not yet fixed
<img width="839" alt="Screenshot 2023-04-20 at 16 38 37" src="https://user-images.githubusercontent.com/103033758/233383864-149f9d49-b732-4b81-b8e1-59bb9a366aaa.png">